cou/utils/juju_utils.py

```diff
@@ -96,7 +96,7 @@ def retry(
     :rtype: Callable
     """
 
-    def _wrapper(func: Callable) -> Callable:
+    def _wrapper(func: Callable) -> Callable:  # pylint: disable=W9011
```

Comment:
Why this disable is needed here?

Reply:
If we remove lint will complain that is missing docstring and I think there isn't much to document about it. Do you think worth adding docstring to this function?

Reply:
It was not there before ant did not failed, that's why I'm asking.

Reply:
Because pylint wasn't checking docstrings for private functions before. With Gabriel's patch here, it is now checking so we need to disable it for this wrapper function..